tools/gcs-bucket-mover/gcs_bucket_mover/cli.py

```diff
@@ -151,8 +150,16 @@ def _get_parsed_args():
     parser.add_argument(
         '--skip_versioning',
         action='store_true',
-        help='Don\'t copy the Versioning setting from the source bucket.')
+        help='Don\'t copy the Versioning setting from the source bucket.') 
+    parser.add_argument(
+        '--retain_source_bucket',
+        #retain_source_bucket=False,
+        choices=[
```

Comment:
Treat retain_source_bucket as flag since its a boolean parameter and use action='store_true'.  By default it will be false and when flag is included then it becomes True